### PR TITLE
Pull Request for Lesson02 Exercises

### PR DESCRIPTION
**Pull request by LouReis**
grid_print_final.py is the file for the first exercise for Grid Print
Fizz_Buzz.py is the file for the second exercise for Fizz Buzz
series.py is the file for the third exercise that contains Fibonacci Series, Lucas Series, & Sum Series

Please review all of these exercises for completion of Lesson02

### CODE REVIEW

**Comment by natasha-aleksandrova** [comment]
please sync your fork; it's showing files from other students.

https://help.github.com/articles/syncing-a-fork/

File: students/LouReis/Lesson03/slicing.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# slicing.py
+#
+#
+# Coder: LouReis
+
+def copy_sequence(seq):
+    """Return a copy of the sequence entered"""
+    return seq
```
Comment (natasha-aleksandrova) [inline students/LouReis/Lesson03/slicing.py]: this isn't returning a copy

File: students/LouReis/Lesson03/slicing.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+#    seq = str(seq)
+#    seq = seq[-2]+seq[2:-2]+seq[1]
+#    return seq.join()
+
+def copy_first_last(seq):
+    """Return the sequence with first and last items entered swapped"""
+    if type(seq) is tuple:
+        return (seq[-1],)+seq[1:-1]+(seq[0],)
+    elif type(seq) is str:
+        return (seq[-1]+seq[1:-1]+seq[0])
```
Comment (natasha-aleksandrova) [inline students/LouReis/Lesson03/slicing.py]: use slicing instead of indexing; if you only use slicing then you don't need to check the data type

**Comment by LouReis** [comment]
Done...
 

Sent: Friday, December 14, 2018 at 2:49 PM

**Comment by LouReis** [comment]
I synced the fork...

 

Thank you

Lou

 

Sent: Friday, December 14, 2018 at 2:53 PM

**Comment by natasha-aleksandrova** [comment]
Perfect, thank you. Please sync the fork every time you submit.

**Pull request merged**